org.scala-ide.sdt.core/src/org/scalaide/ui/internal/editor/decorators/params/byname/CallByNameParamAtCreationHighlightingParticipant.scala

```diff
@@ -0,0 +1,8 @@
+package org.scalaide.ui.internal.editor.decorators.params.byname
+
+import org.scalaide.core.extensions.SemanticHighlightingParticipant
+
+class CallByNameParamAtCreationHighlightingParticipant extends
```

Comment:
I think with this one you win the price for introducing the longest fully qualified name. ;)